src/lib/list/list-option.html

```diff
@@ -0,0 +1,11 @@
+<div [ngClass]="(checkboxPosition == 'before')?'mat-list-item-content' : 'mat-list-item-content-reverse'" >
```

Comment:
Changed '[ngClass]' to '<div [class.mat-list-option-reverse]="checkboxPosition == 'after'">'